infrastructure/sentry-subgraph/src/pool-factory.ts

```diff
@@ -1,4 +1,4 @@
-import { Address, BigInt, ethereum, log } from "@graphprotocol/graph-ts";
+import { Address, bigInt, BigInt, ethereum, log } from "@graphprotocol/graph-ts";
```

Comment:
Are we sure about the small case bigint here ?

Reply:
Removed, it was unused.